src/components/DailyWeather.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { useWeather } from './WeatherProvider';
+import CreateDailyWeather from '../CreateDailyWeather';
+
+export default function DailyWeather() {
+  const { dailyWeather } = useWeather();
+  const [weather, setWeather] = useState([]);
+
+  useEffect(() => {
+    if (dailyWeather) {
+      const { list } = dailyWeather;
+      const dailyWeatherInfo = list.map((weatherData, index) => {
+        const weatherObj = {
+          id: index,
+          timestamp: weatherData.dt,
+          temperature: weatherData.temp.day,
```

Comment:
I think right now the temperature being displayed is a bit ambiguous.  

You can read from the API docs that it's the temp. at noon local time, but that'd probably be unclear to the user.

Reply:
Yeah I realize this can be a bit ambiguous because it doesn't represent the current temperature at the time the search was made but rather noon (which could have already passed). The only problem that I ran into was that the daily weather API call only returns these specific temperature.

 "temp": {
                "day": 63.16,
                "min": 51.46,
                "max": 65.01,
                "night": 51.46,
                "eve": 60.71,
                "morn": 57.34
            },

I think to incorporate the current temperature, I may need to parse the currentWeatherAPI instead of the dailyWeatherAPI? I haven't looked into it yet but will do. Thanks for pointing this out!

Reply:
I think I meant if a user looked at the temperatures for each day without any context, they wouldn't really be able to infer exactly what temperature is being displayed. Like it could be the maximum temperature for that day, but it could also be the minimum or the average daily temperature.   

In this case it's noon, which I personally don't think is clear? Unless that's just something I'm missing. I hope that makes sense.

Reply:
No, I completely understand where you're coming from. I'm just asking for your input and if you have a specific preference on how you think the temperature should to be displayed to be more clear for the end-user. Like do you think it would be better if the temperature included the time of day next to it, a description, etc?

Reply:
Maybe displaying min/max would be most helpful? I'm just looking at different weather sources now:  

weather.com 10 Day Forecast:
<img width="214" alt="image" src="https://github.com/awong-pdx/frontend-weather-dashboard/assets/74124296/7708793c-969e-4b09-9c4a-956279fe79c1">

Apple weather app:
<img width="214" alt="apple weather daily forecast" src="https://github.com/awong-pdx/frontend-weather-dashboard/assets/74124296/c641ee05-d5a7-46f6-a068-16a57c10c6c3">

Reply:
Yeah, it seems like giving them the min and max is probably the most useful. Will do.